### ISSUE TITLE
tslint is not passing.

### ISSUE THREAD

**Issue by morningdew830** [BioPhoton/angular-star-rating#17]
I am trying to use angular-star-rating v2.0.1 for my app but can't pass tslint.

```
/node_modules/angular-star-rating/src/star-rating-struct.ts[1, 31]: " should be '
... ... ...
/node_modules/angular-star-rating/src/star-rating-config.ts[51, 7]: comment must start with a space
/node_modules/angular-star-rating/src/star-rating-config.ts[57, 9]: Identifier 'fractionSize' is never reassigned;
... ... ...
```
will this be fixed soon?

**Comment by BioPhoton** [comment]
As a quick fix you could exclude the lib from linting.
 I will see if I find time in the next days.

**Comment by BioPhoton** [comment]
fixed in version 2.0.3
